docs/getting--started/javascript.md

```diff
@@ -263,6 +263,27 @@ const transaction = await eas.multiRevokeOffchain([data1, data2]);
 await transaction.wait();
 ```
 
+### Verify an Off-chain Attestation
+
+To verify an off-chain attestation, you can use the `verifyOffchainAttestationSignature` function provided by the EAS SDK. Here's an example:
+
+```javascript
+import { OFFCHAIN_ATTESTATION_VERSION, Offchain, PartialTypedDataConfig } from '@ethereum-attestation-service/eas-sdk';
+// import your off chain attestation file
+import attestation from './schema-2-attestation-1690060000.eas.txt';
```

Comment:
Any way to change this to 

const attestation = {Offchain attestation};

Reply:
Sure! Updated